### PR TITLE
docs: added contributor package

### PR DESCRIPTION
**Pull request by phklive**
**Motivation**

Added a Contributor Package to help onboarding newcomers willing to contribute to the project.

**Description**

It helps by providing quick start guide, interesting foundational knowledge resources, issue organisation, pr formats, communication means.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,102 @@ Our aim is to infuse these strengths into the Ethereum consensus client ecosyste
...
+  - [What can I only do in Erlang](https://hackmd.io/ZpUazPomRvacKoyW2vq54g)
+  - [Stacking theory for systems design](https://medium.com/@jlouis666/stacking-theory-for-systems-design-2450e6300689)
+  - [On Erlang States and Crashes](http://jlouisramblings.blogspot.com/2010/11/on-erlang-state-and-crashes.html)
+  - [How Erlang does scheduling](http://jlouisramblings.blogspot.com/2013/01/how-erlang-does-scheduling.html)
+
+With this foundation you should have a basic understanding of the Elixir language and the Erlang VM. You can then start (or in parallel) learning about the Ethereum protocol.
+
+**Learning Ethereum**:
+- **Books**:
+  - [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook)
```
Comment (mpaulucci) [inline README.md:61]: I would not add this one, it seems quite unrelated to the consensus part that we're working here

File: README.md [hunk 2]
```diff
@@ -15,6 +15,102 @@ Our aim is to infuse these strengths into the Ethereum consensus client ecosyste
...
+
+Please adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification when crafting commit messages.
+
+### Communication:
+
+**For specific questions** related to issues, you can use the chat provided under each issue.
+
+**For general questions**, discussions, and feedback, you can use the [Lambda Consensus Client](https://t.me/lambdaconsensus) telegram group.
+
+### Together, let's shape the future of Ethereum! 🚀
```
Comment (mpaulucci) [inline README.md:112]: nice comment, but I would change it to not be a headline, maybe bold instead?

Comment (phklive) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (phklive) [reply README.md]: done

**Review by mpaulucci — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -17,6 +17,101 @@ Our aim is to infuse these strengths into the Ethereum consensus client ecosyste
...
+
+**Then you're in the right place! 🚀**
+
+### Getting Started:
+
+#### 1. **Installation**:
+
+- **Prerequisites**: Before diving in, ensure you have the necessary tools installed. Check out the [Prerequisites](#prerequisites) section for guidance.
+  
+- **Clone the Repository**:
```
[the remark this answers is not in the document]
Comment (phklive) [reply README.md:32]: I think we should leave it here, i link to the installing and running section. 

This is mostly to give a one section guide to running, learning and contributing.

[the remark this answers is not in the document]
Comment (phklive) [reply README.md:32]: I am not adding any info's only re-using

**Comment by mpaulucci** [comment]
Great work! 🚀
